homeassistant/components/zwave_js/api.py

```diff
@@ -452,6 +482,8 @@ def device_registered(device: DeviceEntry) -> None:
         controller.on("inclusion started", forward_event),
         controller.on("inclusion failed", forward_event),
         controller.on("inclusion stopped", forward_event),
+        controller.on("validate dsk and enter pin", forward_dsk),
+        controller.on("grant security classes", forward_requested_grant),
         controller.on("node added", node_added),
```

Comment:
Looks like `lowSecurity` is missing in the forwarded node details on "node added" event. The frontend needs this info to be able to tell the user if a device was added non securely after inclusion.